### PR TITLE
Add optional arg to image-scan-report to support scanning image by digest

### PR DESCRIPTION
**Pull request by cslocum**

### CODE REVIEW

**Review by jaytmiller — COMMENTED** [review]
Aside from the bugs you found in Slack here were some minor things GitHub sits on until you submit them a second time...

**Review by jaytmiller — APPROVED** [review]
Bug fixes you found reporting status look great.  Thanks!

File: tools/image-scan-report [hunk 1]
```diff
@@ -4,7 +4,7 @@
 """This tool pulls down Docker image scan reports from ECR based on
 the current environment which is used to identify the image.
 
-image-scan-report   <Ubuntu version substring, e.g. Focal>   <min severity level, e.g. medium>
+image-scan-report <Ubuntu version substring, e.g. Focal> <min severity level, e.g. medium>
```
[the remark this answers is not in the document]
Comment (cslocum) [reply tools/image-scan-report:7]: Sure

File: tools/image-scan-report [hunk 2]
```diff
@@ -78,14 +83,14 @@ FAIL_LEVELS = [
...
-def get_report_dict(version, levels):
+def get_report_dict(version, levels, image_digest):
     """Get the overall scan report dict which includes both ECR findings which
     have severities in `levels` and the Ubuntu status string for the CVE for
     Ubuntu `version`.
     """
     print("Fetching ECR vulnerability scan",file=sys.stderr)
     sys.stderr.flush()
-    vulnerabilities = get_scan_results()
+    vulnerabilities = get_scan_results(image_digest)
```
[the remark this answers is not in the document]
Comment (cslocum) [reply tools/image-scan-report:93]: Ah!  Yes, thanks for catching that!

**Pull request merged**